src/main/resources/schema.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE id_table(
```

Comment:
Please rename this table to `calendar_event` or `calendar_event_key`.  I prefer the former. Tables should have clear names (i.e. include the name of the entity, etc.).  Although we're not storing any other information about the calendar event here, we may in the future.